packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -1241,19 +1331,19 @@ class Auth {
     /**
      * A wrapper method for commerce-sdk-isomorphic helper: getPasswordLessAccessToken.
      */
-    async getPasswordLessAccessToken(parameters: LoginPasswordlessParams) {
-        const pwdlessLoginToken = parameters.pwdlessLoginToken
+    async getPasswordLessAccessToken(parameters: GetPasswordLessAccessTokenPublicParams) {
```

Comment:
Nit: 

I'd like to suggest not capitalizing the 'L' in Passwordless but we've capitalized it in isomorphic and everywhere else 

```suggestion
    async getPasswordLessAccessToken(parameters: GetPasswordlessAccessTokenPublicParams) {
```

Reply:
Agree, it would have to be a commerce-sdk-isomorphic v5 change since the package has already shipped